common/core/src/commonMain/kotlin/com/trafi/ui/theme/internal/CurrentSpacingScale.kt

```diff
@@ -2,4 +2,12 @@ package com.trafi.ui.theme.internal
 
 class CurrentSpacingScale(
     val globalMargin: Dp,
+
+    val badgeSpacer: Dp,
+
+    val horizontalPaddingMedium: Dp,
+    val horizontalPaddingSmall: Dp,
```

Comment:
Do we expect to have more "medium" and "small" components than badges? Should they all share these same constants? If no, these probably shouldn't be globally theme-able. If yes, we should consider using a "shape" concept like material design uses, where they let SDK users globally override small, medium and large "shapes", where each shape can have its own properties like corner radius, padding, etc.